docs/source/tutorial/package_a_project.md

```diff
@@ -49,7 +68,7 @@ main(
 
 This is equivalent to running `kedro run`, and you can provide all the parameters described by `kedro run --help`.
 
-### Docker, Airflow and Deployment
+### Docker, Airflow and other deployment targets
 
```

Comment:
We should push the open source community here e.g. kedro-vertexai or kedro-argo etc.